src/binary_tree/binary_avl_tree.h

```diff
@@ -0,0 +1,19 @@
+#ifndef DS_AVL_TREE_
+#define DS_AVL_TREE_
+#include <stddef.h>
+
+typedef struct dsAVLNode {
+	size_t height;
+	void *value;
+	struct dsAVLNode *left;
+	struct dsAVLNode *right;
+} dsAVLNode_t;
+
+
+typedef struct dsAVLTree {
+	dsAVLNode_t *head;
+	size_t count;
```

Comment:
@ViniciusA1 Could you please change 'count' to 'size', please? seems more intuitive